src/components/Register/Register.tsx

```diff
@@ -0,0 +1,597 @@
+                  />
+                </Stack>
+              </LocalizationProvider>
+            </div>
+
+            <div className="form-input__register">
+              <Typography
+                align="left"
+                gutterBottom
+                sx={{
+                  fontSize: '16px',
+                  lineHeight: '24px'
+                }}>
+                Giới tính
+                <Typography
+                  sx={{
+                    marginLeft: '2px',
+                    display: 'inline-block',
+                    color: 'red'
+                  }}>
+                  (*)
+                </Typography>
+              </Typography>
+              <Controller
+                name="gender"
+                control={control}
+                render={({ field, fieldState: { error } }) => (
+                  <FormControl>
+                    <RadioGroup
+                      sx={{ mb: 2 }}
+                      row
+                      aria-labelledby="demo-controlled-radio-buttons-group"
+                      {...field}
+                      onChange={(event) => {
+                        field.onChange(event.target.value);
+                      }}>
+                      <FormControlLabel
+                        value="nale"
+                        control={<Radio />}
+                        label="Nam"
+                      />
+                      <FormControlLabel
+                        value="female"
+                        control={<Radio />}
+                        label="Nữ"
+                      />
+                    </RadioGroup>
+                    {error && (
+                      <FormHelperText sx={{ color: 'red' }}>
+                        {error?.message}
+                      </FormHelperText>
+                    )}
+                  </FormControl>
+                )}
+              />
+            </div>
+
+            <div className="form-input__register">
+              <Typography
+                align="left"
+                gutterBottom
+                sx={{
+                  fontSize: '16px',
+                  lineHeight: '24px'
+                }}>
+                Tỉnh/Thành phố
+                <Typography
+                  sx={{
+                    marginLeft: '2px',
+                    display: 'inline-block',
+                    color: 'red'
+                  }}>
+                  (*)
+                </Typography>
+              </Typography>
+              <Controller
+                name="province_id"
+                control={control}
+                render={({ field, fieldState: { error } }) => (
+                  <FormControl fullWidth required sx={{ mb: 2 }}>
+                    <InputLabel id="input-label">Province</InputLabel>
+                    <Select
+                      labelId="demo-simple-select-label"
+                      id="province-select"
+                      label="province"
+                      {...field}
```

Comment:
```suggestion
                    <Select
                      defaultValue=""
                      labelId="demo-simple-select-label"
                      id="province-select"
                      label="province"
                      {...field}
```

Sửa lại labelId, thêm defaultValue='' để tránh lỗi out of range